telco-core/configuration/reference-crs-kube-compare/required/networking/sriov/SriovOperatorConfig.yaml

```diff
@@ -0,0 +1,13 @@
+# required
+# count: 1
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  configDaemonNodeSelector:
+    node-role.kubernetes.io/worker: ""
```

Comment:
I know it's in the RDS, but should this be enforce on all workers? Some customers will use different selectors

Reply:
+1

Reply:
Selector should be user variable.